src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -186,18 +189,28 @@ public function editAction(int $orderStateId, Request $request)
             $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
         }
 
+        $editableOrderState = $this->getQueryBus()->handle(new GetOrderStateForEditing((int) $orderStateId));
+
         return $this->render('@PrestaShop/Admin/Configure/ShopParameters/OrderStates/edit.html.twig', [
             'orderStateForm' => $orderStateForm->createView(),
             'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
-            'editableOrderState' => $this->getQueryBus()->handle(new GetOrderStateForEditing((int) $orderStateId)),
+            'editableOrderState' => $editableOrderState,
             'contextLangId' => $this->getContextLangId(),
             'templatesPreviewUrl' => _MAIL_DIR_,
+            'enableSidebar' => true,
             'languages' => array_map(
                 function (array $language) {
                     return [
                         'id' => $language['iso_code'],
                         'value' => sprintf('%s - %s', $language['iso_code'], $language['name']), ];
                 }, $this->get('prestashop.adapter.legacy.context')->getLanguages()),
+            'layoutTitle' => $this->trans(
+                'Editing status %name%',
```

Comment:
```suggestion
                'Editing order state %name%',
```

I feel like we're missing something with status vs. state, it's almost the same but we should probably use one word?

Reply:
Daniel's suggestion is easier to scan. We're already on the order status page, so it's clear enough.

Reply:
Status ✅

Reply:
Editing order status %name%